spring-cloud-sleuth-samples/spring-cloud-sleuth-sample-zipkin/pom.xml

```diff
@@ -107,6 +107,13 @@
 			<artifactId>zipkin-junit</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<!-- otherwise spring boot's version of okhttp kicks zipkin-junit's deps out of alignment -->
```

Comment:
didn't know a better way of doing this, as I didn't want to affect the root classpath on account of a junit rule :P